Broken build in master branch

master branch cannot be built (as of head 72599226466dea69ebc7cb20d96ab9f78f9fd84d)
```

~/sources-home/OpenFoodFacts-androidApp/app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/IngredientsProductFragment.java:148: error: variable tagWithoutLocale is already defined in method onViewCreated(View,Bundle)
              String tagWithoutLocale = tag.replaceAll("(en:|fr:)", "").toUpperCase();
                     ^
~/sources-home/OpenFoodFacts-androidApp/app/src/main/java/openfoodfacts/github/scrachx/openfood/views/MainActivity.java:253: error: cannot find symbol
          public void onClick(MaterialDialog dialog, DialogAction which) {
                                                     ^
symbol: class DialogAction
~/sources-home/OpenFoodFacts-androidApp/app/src/main/java/openfoodfacts/github/scrachx/openfood/views/MainActivity.java:255: error: cannot find symbol
                      Toast.LENGTH_SHORT).show();
                      ^
symbol: variable Toast
~/sources-home/OpenFoodFacts-androidApp/app/src/main/java/openfoodfacts/github/scrachx/openfood/views/MainActivity.java:254: error: cannot find symbol
              Toast.makeText(getApplicationContext(),"Cancelled",
              ^
symbol: variable Toast
~/sources-home/OpenFoodFacts-androidApp/app/src/main/java/openfoodfacts/github/scrachx/openfood/views/MainActivity.java:247: error: cannot find symbol
          public void onClick(MaterialDialog dialog, DialogAction which) {
                                                     ^
symbol: class DialogAction
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
5 errors
```

--- a/app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/IngredientsProductFragment.java
+++ b/app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/IngredientsProductFragment.java
@@ -145,7 +145,6 @@ public class IngredientsProductFragment extends BaseFragment {
 
             for (String tag : product.getAdditivesTags()) {
                 String tagWithoutLocale = tag.replaceAll("(en:|fr:)", "").toUpperCase(Locale.getDefault());
-                String tagWithoutLocale = tag.replaceAll("(en:|fr:)", "").toUpperCase();
                 final List<Additive> la = mAdditiveDao.queryBuilder().where(AdditiveDao.Properties.Code.eq(tagWithoutLocale.toUpperCase())).list();
                 additiveProduct.append(getSpanTag(tagWithoutLocale, view));
                 //Display additives list with full name

--- a/app/src/main/java/openfoodfacts/github/scrachx/openfood/views/MainActivity.java
+++ b/app/src/main/java/openfoodfacts/github/scrachx/openfood/views/MainActivity.java
@@ -25,6 +25,7 @@ import android.util.Log;
 import android.view.Menu;
 import android.view.MenuItem;
 
+import com.afollestad.materialdialogs.DialogAction;
 import com.afollestad.materialdialogs.MaterialDialog;
 import com.mikepenz.fastadapter.commons.utils.RecyclerViewCacheUtil;
 import com.mikepenz.google_material_typeface_library.GoogleMaterial;
@@ -41,6 +42,7 @@ import com.mikepenz.materialdrawer.model.SectionDrawerItem;
 import com.mikepenz.materialdrawer.model.interfaces.IDrawerItem;
 import com.mikepenz.materialdrawer.model.interfaces.IProfile;
 import android.support.v7.app.AlertDialog;
+import android.widget.Toast;
 
 import butterknife.BindView;
 import openfoodfacts.github.scrachx.openfood.BuildConfig;

